charts/keda-kaito-scaler/templates/keda-kaito-scaler.yaml

```diff
@@ -88,8 +79,6 @@ spec:
             - --v={{ .Values.log.level }}
             - --metrics-port={{ .Values.ports.metrics }}
             - --health-probe-port={{ .Values.ports.healthProbe }}
-            - --webhook-port={{ .Values.ports.webhook }}
-            - --working-namespace={{ .Release.Namespace }}
```

Comment:
I think `--working-namespace` should be kept, so user can configure specified namespace for keda-kaito-scaler component.

Reply:
good catch, reverted.